### PR TITLE
docs: Update docs for rrule/type

### PR DESCRIPTION
**Pull request by mikebryant**
I got a bit stuck getting this to work until I read the source. Updating the docs here to make it clear how to use rrule

### CODE REVIEW

**Review by buraizu — CHANGES_REQUESTED** [review]
Small edit for consistency

File: docs/resources/downtime.md [hunk 1]
```diff
@@ -72,12 +72,12 @@ resource "datadog_downtime" "foo" {
 
 Required:
 
-- **type** (String) One of `days`, `weeks`, `months`, or `years`
+- **type** (String) One of `days`, `weeks`, `months`, `years`, or `rrule`
```
[the remark this answers is not in the document]
Comment (mikebryant) [reply docs/resources/downtime.md]: Cheers, done :)

**Review by buraizu — DISMISSED** [review]

**Comment by skarimo** [comment]
Could you rebase on master and regenerate new docs. Thanks!

**Comment by mikebryant** [comment]
@skarimo done :)

**Pull request merged**